generators/frontend/templates/react-demo/components/CategoryCard.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import Link from 'next/link'
+import Util from '../common/Util'
+import styled from 'styled-components'
+
+export default ({ id, name }) => {
+
+  const Wrapper = styled.div`
+    color: darkslategrey;
+    margin: 40px 20px auto 20px;
+    width: 14em;
+    border-color: transparent;
+    height: 10em;
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    cursor: pointer;
+    background-color: whitesmoke;
+    div {
+      padding: 20px;
+      font-size: 16px;
+      text-transform: uppercase;
+      background-color: rgba(255,255,255, 0.9);
+    }
+    :first-child {
```

Comment:
I attached you a screenshot. I hardcoded the images because the API doesn't provide an image for each category but only for the products. This is a bad practice but I did it because it's only a demo :)

Reply:
<img width="739" alt="screen shot 2018-11-09 at 12 52 25" src="https://user-images.githubusercontent.com/23245590/48259187-e280a180-e41f-11e8-8171-182c15b995ea.png">